go/libkb/id_table.go

```diff
@@ -1127,7 +1127,9 @@ func (idt *IdentityTable) populate() (err error) {
 			idt.G().Log.Debug("Ignoring bad chain link with sig ID %s: %s", link.GetSigID(), reason)
 			continue
 		}
-
+		if link.IsStubbed() {
+			continue
```

Comment:
Should there be an assert here that the type isn't something we expected to have a full link for?